### DISCUSSION TITLE
Near-term vetting: Nuclear

### DISCUSSION THREAD

**Discussion by pweigmann** [pik-piam/mrremind#540] [Near-term validation]
**Variable: `Capacity|Electricity|Nuclear`**

**Sources:**

[IAEA-PRIS](https://pris.iaea.org/PRIS/CountryStatistics/CountryStatisticsLandingPage.aspx)
* country pipeline data
* available project status: "operational", "construction"

[IAEA Estimates](https://www-pub.iaea.org/MTCD/Publications/PDF/RDS-1-44_web.pdf)
* using global estimates for 2030

[Ember](https://ember-climate.org/data-catalogue/yearly-electricity-data/)
* historic country data, average of 2018 - 2022 used for 2020 period

**Assumptions + data preparation:**

* slow technology: potential new power plants that are not yet under construction won't be operational in 2030

`max` thresholds only:
* power plants under construction will be finished in 2030
* no decommissioning

**Threshold definitions:**

**2020**
* `min_red` global: -20% (min(Ember, PRIS-"operational")) ; regional: -40% (min(Ember, PRIS-"operational"))
* `min_yel` global:  -10% (min(Ember, PRIS-"operational"))   ; regional: -20% (min(Ember, PRIS-"operational"))
* `max_yel` global: +10% (max(Ember, PRIS-"operational"))  ; regional: +20% (max(Ember, PRIS-"operational"))
* `max_red` global: +20% (max(Ember, PRIS-"operational")) ; regional: +40% (max(Ember, PRIS-"operational"))

**2030**
* `min_red` global: -10% (PRIS-"operational" * 0.8) ; no regional
* `min_yel` global:  -5% (PRIS-"operational" * 0.9)     ; regional: -40% (PRIS-"operational" * 0.8) 
* `max_yel` global: +5% (PRIS-"operational" + PRIS-"inactive (JPN)" + PRIS-"construction" * 0.75)   ; 
regional: +40% (PRIS-"operational" + PRIS-"inactive (JPN)" + PRIS-"construction") 
* `max_red` global: +10% (461 GW (high IAEA 2030 estimate)) ; no regional

**Data Sources**

**IAEA-PRIS**: https://pris.iaea.org/PRIS/CountryStatistics/CountryStatisticsLandingPage.aspx (2024-08-08)
**IAEA Estimates**: https://www-pub.iaea.org/MTCD/Publications/PDF/RDS-1-44_web.pdf
**Ember**: https://ember-climate.org/data-catalogue/yearly-electricity-data

**Change log**
- Update 08.2025: updated to IAEA-PRIS database state as of 2025-08-07.
- Update 06.2026: updated to IAEA-PRIS database state as of 2026-06-08.

**Comment by RahelMA** [comment]
The global boundaries for 2020 are set more generous, as nuclear power plants from Japan can faster be switched on and off, making it more difficult for models to capture `Capacity|Electricity|Nuclear`.

**Reply by gunnar-pik** [reply]
Japan has substantial capacity that is "suspended", but in principle can be reactivated. Not sure how this is dealt with in the PRISM database - they might be misleadingly labeled as permanently shut down. We should allow for account for possible reactivation of these dormant capacities by making "max_red" and "max_yel"  also for 2030 more lenient.

**Reply by gunnar-pik** [reply]
Just saw that IAEA published a [report ](https://www.iaea.org/publications/15756/energy-electricity-and-nuclear-power-estimates-for-the-period-up-to-2050) recently putting the high end of 2030 capacities at 461 TW. They don't give an explanation, and I also don't find this overly plausible. But since this is a relevant authority, we still may want base our max_red on this, plus some tolerance (@pweigmann, @RahelMA)

**Reply by pweigmann** [reply]
@gunnar-pik I updated the original post to reflect the changes we talked about. We now use Ember or PRIS data for 2020, whichever is lower (min thresholds) or higher (max thresholds). Also for the global max_red threshold in 2030 we use the ``high`` estimate by IAEA of 461 GW plus 10% buffer.

It is nice that we are refining these thresholds to be more useful, on the other hand, the calculations get complicated quite quickly as well and with every exception and special case we also might lose some transparency.

Edit: Adjustments for Japan still missing.

**Comment by EHooijschuur** [comment]
The data that I received a while ago, has a remarkable drop in 2023. If that is still there after the updates discussed above, do we know where that comes from? Maybe data from a specific country is not available that year?
